document/core/syntax/instructions.rst

```diff
@@ -327,6 +329,13 @@ this action conditionally, if the read value is equal to a provided comparison
 argument. All other :ref:`atomicops <syntax-atomicop>` have behavior of the
 :ref:`ibinop <syntax-ibinop>` of the same name.
 
+The |WAKE| and |WAIT| instructions provide primitive synchronization between
+:ref:`threads <syntax-thread>`. The |WAIT| instructions atomically load a value
+from the calculated *effective address* and compare it to an expected value. If
```

Comment:
Nit: no need to emphasize "effective address" here, since the defining occurrence already was above.